skycultures/tibetan/asterism_names.eng.fab

```diff
@@ -0,0 +1,28 @@
+# Temporary solution for Tibetan Lunar Mansions
+L00 _("LM00 Takar",    "asterism")
```

Comment:
Is the serial number of the asterism important here? I guess it shouldn't be translatable.

Reply:
It is meant to be "Lunar Mansion Zero". I regard it important enough to be shown. Current display settings don't allow showing the number in another way.

Reply:
OK

Reply:
Please add `# TRANSLATORS:` notes for translators about "Lunar Mansion Zero" strings

Reply:
done.